Documentation/Signoffs/Safety_System.md

```diff
@@ -0,0 +1,152 @@
+# Safety System
+
+# Subsystem Function:
+The sole purpose of this system is to secure the safety of both the users and the operators of the device. The primary functions of the system come from cabinet restrictions, modified chassis, and informational stickers. Another focus of this system is to meet all the requirement of having a piece of electrical machinery. These standards are mostly taken from OSHA's Electrical General Requirements page.
+
+
+
+# Constraints:
```

Comment:
Where are these standards coming from and why are they here? They aren't relevant.